source/main.cpp

```diff
@@ -130,7 +137,8 @@ int main(int argc, char *argv[])
 	catch(const runtime_error &error)
 	{
 		Audio::Quit();
-		GameWindow::ExitWithError(error.what());
+		bool doPopUp = testToRun.empty();
```

Comment:
I think it's fine to show the user the error dialog if they are running a test manually

Reply:
There is no parameter that tells ES if the test was started from a bash-script or if it was started manually by an user directly from the command-line.